manifests/m/Microsoft/UrlRewrite/2.1/Microsoft.UrlRewrite.installer.yaml

```
@@ -0,0 +1,260 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.UrlRewrite
+PackageVersion: '2.1'
+InstallerType: wix
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+InstallationMetadata:
+  DefaultInstallLocation: .\inetsrv
+Dependencies:
+  WindowsFeatures:
+  # IIS 7.0 CoreWebEngine and W3SVC should be installed and enabled.
+  - IIS-WebServer
```

```suggestion
  - NetFx3
  - IIS-WebServerRole
  - IIS-WebServer
```

- See https://github.com/microsoft/winget-pkgs/pull/184181#issuecomment-2429257816